<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="5" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-22T13:05:27 EST" name="Suite" finished-at="2024-02-22T13:06:03 EST" duration-ms="35414">
    <groups>
    </groups>
    <test started-at="2024-02-22T13:05:27 EST" name="Test" finished-at="2024-02-22T13:06:03 EST" duration-ms="35414">
      <class name="MyAtomation.SeleniumAutomation.Tests.TestngTests1">
        <test-method is-config="true" signature="launchAppPage()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:28 EST" name="launchAppPage" finished-at="2024-02-22T13:05:33 EST" duration-ms="5179" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchAppPage -->
        <test-method signature="test1(java.util.HashMap)[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:33 EST" name="test1" data-provider="myDataProvider" finished-at="2024-02-22T13:05:51 EST" duration-ms="17716" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=*r#NHQH59V6WL@Q, email=sainikil96@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:51 EST" name="closeBrowser" finished-at="2024-02-22T13:05:51 EST" duration-ms="88" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="launchAppPage()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:51 EST" name="launchAppPage" finished-at="2024-02-22T13:05:57 EST" duration-ms="6346" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchAppPage -->
        <test-method retried="true" signature="test2()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:57 EST" name="test2" finished-at="2024-02-22T13:05:57 EST" duration-ms="533" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or passwor.] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or passwor.] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at MyAtomation.SeleniumAutomation.Tests.TestngTests1.test2(TestngTests1.java:84)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:57 EST" name="closeBrowser" finished-at="2024-02-22T13:05:58 EST" duration-ms="82" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="launchAppPage()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:05:58 EST" name="launchAppPage" finished-at="2024-02-22T13:06:02 EST" duration-ms="4330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchAppPage -->
        <test-method signature="test2()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:06:02 EST" name="test2" finished-at="2024-02-22T13:06:02 EST" duration-ms="511" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or passwor.] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or passwor.] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at MyAtomation.SeleniumAutomation.Tests.TestngTests1.test2(TestngTests1.java:84)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:249)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests1@41e36e46]" started-at="2024-02-22T13:06:03 EST" name="closeBrowser" finished-at="2024-02-22T13:06:03 EST" duration-ms="82" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- MyAtomation.SeleniumAutomation.Tests.TestngTests1 -->
      <class name="MyAtomation.SeleniumAutomation.Tests.TestngTests2">
        <test-method is-config="true" signature="launchAppPage()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests2@6e171cd7]" started-at="2024-02-22T13:05:28 EST" name="launchAppPage" finished-at="2024-02-22T13:05:33 EST" duration-ms="5061" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchAppPage -->
        <test-method signature="test3()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests2@6e171cd7]" started-at="2024-02-22T13:05:33 EST" name="test3" finished-at="2024-02-22T13:05:33 EST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:MyAtomation.SeleniumAutomation.Tests.TestngTests2@6e171cd7]" started-at="2024-02-22T13:05:33 EST" name="closeBrowser" finished-at="2024-02-22T13:05:33 EST" duration-ms="86" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- MyAtomation.SeleniumAutomation.Tests.TestngTests2 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
